docs/source/using/profiles.adoc

```diff
@@ -63,4 +85,33 @@ To delete a profile, run:
 $ minishift profile delete PROFILE_NAME
 ----
 
-NOTE: The default profile 'minishift'  can not be deleted.
+NOTE: The default profile *minishift* cannot be deleted.
+
+[[example-workflows-profiles]]
+== Example Workflows
+
+To create a new profile *foo* with specific configurations, run:
```

Comment:
@LalatenduMohanty you want to keep "foo"?

Reply:
We need to explain what's happening here. Here we use the the set command to switch to the new profile, making it the active one. Then we execute a series of commands against the active profile.